documentation/asciidoc/computers/compute-module/cm-emmc-flashing.adoc

```diff
@@ -1,114 +1,104 @@
 [[flash-compute-module-emmc]]
 == Flash an image to a Compute Module
 
+TIP: To flash the same image to multiple Compute Modules, use the https://github.com/raspberrypi/rpi-sb-provisioner[Raspberry Pi Secure Boot Provisioner]. To customise an OS image to flash onto those devices, use https://github.com/RPi-Distro/pi-gen[pi-gen].
+
 [[flashing-the-compute-module-emmc]]
 
 The Compute Module has an on-board eMMC device connected to the primary SD card interface. This guide explains how to flash (write) an operating system image to the eMMC storage of a single Compute Module.
 
-NOTE: **Lite** variants of Compute Modules do not have on-board eMMC. Instead, you can follow the procedure to flash a storage device for other Raspberry Pi devices at xref:../computers/getting-started.adoc#installing-the-operating-system[Install an operating system].
-
-When flashing an image to multiple Compute Modules, consider using the Compute Module Provisioner instead:
-
-[.whitepaper, title="Mass Provision with the Compute Module Provisioner", subtitle="", link=https://pip.raspberrypi.com/categories/685-whitepapers-app-notes/documents/RP-003468-WP/Using-the-Compute-Module-Provisioner.pdf]
-****
-The Compute Module Provisioner is a web application that helps developers program many Compute Module devices simultaneously.
-
-It provides a database of kernel images and the ability to run scripts during the flashing process, in addition to automated label printing and firmware updating.
-****
+**Lite** variants of Compute Modules do not have on-board eMMC. Instead, follow the procedure to flash a storage device for other Raspberry Pi devices at xref:../computers/getting-started.adoc#installing-the-operating-system[Install an operating system].
 
 === Prerequisites
 
 To flash the Compute Module eMMC, you need the following:
 
-* Another computer, referred to in this guide as the *host device*. You can use Linux (Raspberry Pi OS or Ubuntu), Windows, or macOS.
+* Another computer, referred to in this guide as the *host device*. You can use Linux (we recommend Raspberry Pi OS or Ubuntu), Windows, or macOS.
 * The Compute Module IO Board xref:compute-module.adoc#io-board-compatibility[that corresponds to your Compute Module model].
-* A micro USB cable.
+* A micro USB cable, or a USB-C cable for Compute Module models since CM5.
```

Comment:
I guess the cable required is actually determined by the IO board, rather than the CM itself?